test/controllers/hosts_controller_test.rb

```diff
@@ -1345,7 +1345,7 @@ class Host::Valid < Host::Managed; end
 
   test "#host update shouldn't diassociate from VM" do
     hostgroup = FactoryBot.create(:hostgroup, :with_environment, :with_subnet, :with_domain, :with_os)
-    compute_resource = compute_resources(:ovirt)
+    compute_resource = compute_resources(:one)
     compute_resource.update(:locations => hostgroup.locations, :organizations => hostgroup.organizations)
```

Comment:
why did you change :ovirt to :one?

Reply:
Because I understood that the purpose of the test is to check that the host update does not disassociate from VM and therefore it does not matter which compute resource to check. also, I noticed that all the other tests here are working with this compute resource compute_resources(:one) and not with ovirt.
If I misunderstood please let me know and I will add the quota and client part so the test works as it is.

Reply:
It shouldn't matter, But I'm wondering what error did you get the cause change the CR? and how these two compute resources are different?

Reply:
I received this error:
Error:
HostsControllerTest#test_0083_#host update shouldn't diassociate from VM:
OvirtSDK4::ConnectionError: Can't send request: Couldn't resolve host name
    app/models/compute_resources/foreman/model/ovirt.rb:522:in `client'
    app/models/compute_resources/foreman/model/ovirt.rb:494:in `validate_quota'
    test/controllers/hosts_controller_test.rb:1349:in `block in <class:HostsControllerTest>'
The two compute resources are different because compute_resources(:ovirt) is an ovirt cr and compute_resources(:one) is a libvirt cr.

Reply:
@yifatmakias, 
It seems like it was on ovirt on purpose.

Reply:
@shiramax Ok so I will change it back to ovirt and deal with the error.